### PR TITLE
Create bellmanford.go

### PR DESCRIPTION
**Pull request by ujjwaltiwari13**
Added bellmanford algorithm implementation in Go

Fixes https://github.com/jainaman224/Algo_Ds_Notes/issues/2202

#### Checklist:
- [X] 4 space indentation.
- [X] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [X] Sample Input / Output is added at the end of file.
- [X] Logic Documentation (Comments).
- [X] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added implementation of bellmanford algorithm in Golang.

#### Languages Used:
- Go

#### Files Added:
- bellmanford.go

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by ujjwaltiwari13** [comment]
@jainaman224  please review the PR which includes bellmanford algorithm implementation in golang as assigned.

**Comment by ujjwaltiwari13** [comment]
I have given proper indentation, please review it. @asha15

File: Bellmanford_Algorithm/bellmanford.go [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+func main() {
+
+	
+
```
Comment (AdityaShD) [inline Bellmanford_Algorithm/bellmanford.go]: Please use 4 space indentation instead of tabs

File: Bellmanford_Algorithm/bellmanford.go [hunk 2]
```diff
@@ -0,0 +1,429 @@
+package main
+
+import (
+	"container/list"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
```
Comment (asha15) [inline Bellmanford_Algorithm/bellmanford.go]: keep four space indentation

File: Bellmanford_Algorithm/bellmanford.go (hunk 1 above)
Comment (asha15) [reply Bellmanford_Algorithm/bellmanford.go]: remove the unnecessary lines

File: Bellmanford_Algorithm/bellmanford.go [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+func main() {
+
+	
+
+	str1 := `
+S|A,7|B,6
+A|C,-3|T,9
+B|A,8|T,-4|C,5
+C|B,-2
+T|S,2|C,7
```
Comment (asha15) [inline Bellmanford_Algorithm/bellmanford.go]: keep the indentation 
keep space after comma

File: Bellmanford_Algorithm/bellmanford.go [hunk 4]
```diff
@@ -0,0 +1,429 @@
...
+`
+	G1 := ConstructDirectedGraph(str1)
+	result_list1 := G1.BellmanFordShortestPath(G1.GetVertexByID("S"), G1.GetVertexByID("T"))
+
+	fmt.Print("Graph 01 Bellman-Ford: ")
+	for vtx := result_list1.Front(); vtx != nil; vtx = vtx.Next() {
+		fmt.Printf("%v(=%v) → ", vtx.Value.(*Vertex).ID, vtx.Value.(*Vertex).timestamp_d)
```
Comment (asha15) [inline Bellmanford_Algorithm/bellmanford.go]: see you used tab here instead keep four spaces

File: Bellmanford_Algorithm/bellmanford.go [hunk 5]
```diff
@@ -0,0 +1,429 @@
...
+	}
+
+	println()
+
+	for vtx := G1.GetVertexList().Front(); vtx != nil; vtx = vtx.Next() {
+		fmt.Printf("%v(=%v) \n", vtx.Value.(*Vertex).ID, vtx.Value.(*Vertex).timestamp_d)
+	}
+	println()
+
+
```
Comment (asha15) [inline Bellmanford_Algorithm/bellmanford.go]: remove the unnecessary lines

**Comment by somya-kapoor** [comment]
Due to no activity for more than 7 days, I am closing this PR.

**Pull request closed without merging**